ws-nextjs-app/pages/[service]/watch/[id]/LiveTvPageLayout.tsx

```diff
@@ -22,8 +22,51 @@ export default function LiveTvLayout({ pageData }: LiveTVPageProps) {
     pageData = liveTvFixture;
   }
 
+  const renderCuration = ({ curation }: { curation: CurationType }) => {
+    const {
+      summaries,
+      curationId,
+      title: curationTitle,
+      link,
+      position,
+      ...curationProps
+    } = curation;
+    return (
+      <React.Fragment key={`${curationId}-${position}`}>
+        <Curation
+          summaries={summaries || []}
+          title={curationTitle}
+          position={position}
+          link={link}
+          renderVisuallyHiddenH2Title={position === 0}
+          curationId={curationId}
+          {...curationProps}
+        />
+      </React.Fragment>
+    );
+  };
```

Comment:
This function and the `getSynopses` function can be moved outside and above the `export default function LiveTvLayout` render function, just so the functions aren't re-rendered.